### ISSUE TITLE
Got NaN in prototype

### ISSUE THREAD

**Issue by adaxidedakaonang** [HieuPhan33/REMINDER#10]
When I ran the 15-1 training code, I got the "NaN in prototype" error.  Temporally I have no idea to solve it. Does anyone meet this problem ?

**Comment by HieuPhan33** [comment]
Hi, can you share more details about the problem: what is your training script? At which step and which line of code you encounter the error?

**Comment by adaxidedakaonang** [comment]
Thank you for the reply, I have found the solution by looking for the related issue of repo: PLOP. Detaily you had a divided by zero problem in loss function, which has been found in the repo PLOP.
